adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h

```diff
@@ -37,4 +37,8 @@
 
 @property(nonatomic, copy) NSString *_Nullable playingPlacement;
 
+@property(nonatomic, copy) NSNumber *_Nullable orientations;
+
+@property(nonatomic, copy) NSString *_Nonnull UUID;
```

Comment:
This must be readonly so nobody can change it.

Reply:
@ericleich I thought pubs could set the value of `UUID` to differentiate Banners withe same unitId. Got it, I will update `UUID` as a `readonly` property. Thanks a lot for the instruction. 🙂

Reply:
@YueVungle my original proposal was something that publishers set, but your idea of the adapter automatically generating a UUID is a better way of guaranteeing that every instance of extras has a unique identifier. So publishers no longer need to effectively figure out for themselves how to provide a unique ID. :)

Reply:
@ericleich Got it, thanks a lot for the detailed explanation. ^_^

Reply:
@ericleich  Done, thanks. 😊